dev/RadioButtons/RadioButtons.cpp

```diff
@@ -547,6 +561,58 @@ void RadioButtons::UpdateSelectedItem()
     }
 }
 
+winrt::ToggleButton RadioButtons::TryGetRadioButtonFromRepeaterElement(const winrt::UIElement& element)
+{
+    if (auto const elementAsGrid = element.try_as<winrt::Grid>())
+    {
+        if (elementAsGrid.Children().Size() == 1)
```

Comment:
>== [](start = 44, length = 2)

should we allow size >= 1?

Reply:
Since we would still just grab the first child of the repeater element would that make much sense here? Apparently, it would be a design choice if the wrapping parent has more than one child, but we wouldn't actually pay attention to that and still just treat its first child as relevant here.

If we leave it as it is right now, we would return null and thus basically say we don't know how to treat such a situation (which we wouldn't given the current implementation).

Reply:
We expect it but don't require it right? I prefer the code behind to impose a little restrictions to the visual tree as possible.

Reply:
Yes. It is not required. Will change it to accept values > 1.